test/screenshot/golden.json

```diff
@@ -0,0 +1,4 @@
+{
+  "temporary-package/index.html": "57de91e42bd22a845074bae80f71eca3902ab7c6719f129960bc9018c79db95a",
+  "temporary-package/foo.html": "91f0795700eaba345092dd7507e046daa0d33e1136bad7992011ab454ddb1faf"
+}
```

Comment:
Nit: Add trailing newline to end of file (and probably update your editor config to do this automatically)